src/main/java/org/jabref/logic/importer/fetcher/IsbnViaOttoBibFetcher.java

```diff
@@ -58,6 +58,13 @@ public URL getUrlForIdentifier(String identifier) throws URISyntaxException, Mal
             throw new FetcherException("Could not ", e);
         }
         Element textArea = html.select("textarea").first();
+
+        // inspect the "no results" error message (if there is one)
+        Element potentialErrorMessageDiv = html.select("div#flash-notice.notice.add-bottom").first();
+        if (potentialErrorMessageDiv.hasText() && potentialErrorMessageDiv.text().contains("No Results")) {
+            LOGGER.error("ISBN " + identifier + " not found at ottobib");
```

Comment:
```suggestion
            LOGGER.error("ISBN { } not found at ottobib", identifier);
```
This is the recommended way